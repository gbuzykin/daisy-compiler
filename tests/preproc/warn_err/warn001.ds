
#warning This is a warning!
