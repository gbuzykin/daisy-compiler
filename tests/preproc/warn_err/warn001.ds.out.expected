./preproc/warn_err/warn001.ds:2:2: warning: This is a warning!
 2 | #warning This is a warning!
   |  ^~~~~~~
./preproc/warn_err/warn001.ds: info: warnings 1, errors 0
